src/main/java/controller/CLotto.java

```diff
@@ -0,0 +1,54 @@
+package controller;
+
+import model.LottoNum;
+import model.Lotto;
+import model.WinCheck;
+import valueObject.ErrorMessage;
+import view.VLotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CLotto {
+    private LottoNum lottoNum;
+    private Lotto mLotto;
+    private WinCheck winCheck;
+    private VLotto vLotto;
+    private ErrorMessage errorMessage;
+
+    public CLotto(LottoNum lottoNum, VLotto vLotto) {
+        this.lottoNum = lottoNum;
+        this.vLotto = vLotto;
+    }
+
+    public void run() {
+        int money = vLotto.getMoneyInput();
+        int lottoCount = lottoNum.buyLotto(money);
+        vLotto.displayLottoCount(lottoCount);
+
+        List<List<Integer>> userLottos = new ArrayList<>();
+
+        for (int i = 0; i < lottoCount; i++) {
+            List<Integer> lottoNumbers = lottoNum.lottoNumbers();
+            vLotto.displayLottoNumber(lottoNumbers);
+            userLottos.add(lottoNumbers);
+        }
+
+
+        try {
+            // 당첨 번호 및 보너스 번호 입력받기
+            String winningNumber = vLotto.getWinningNumber();
+            List<Integer> winningNumberList = lottoNum.splitNum(winningNumber);
```

Comment:
List와 같이 자료형의 이름을 변수명에 쓴다면, 나중에 자료형이 변경되었을 때 해당 변수명도 같이 변경되어야 하고 그 변수를 사용하는 다른 곳에도 영향이 끼치기에 자료형은 복수형으로 사용하는 것이 좋습니다.
```suggestion
            List<Integer> winningNumbers = lottoNum.splitNum(winningNumber);
```